modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -836,6 +853,22 @@ func (h *HTTPTransport) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 	return nil
 }
 
+// parseHealthHeaders parse health_headers block
+func parseHealthHeaders(d *caddyfile.Dispenser) (http.Header, error) {
```

Comment:
I don't think I see any value in breaking this out to a separate function. You can inline this.

Reply:
ok, I will fix it immediately